### PR TITLE
more install solutions

### PR DESCRIPTION
**Pull request by ildyria**

### CODE REVIEW

File: docs/md/distributions.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+Their primary goal is to provide easy-to-use and streamlined Docker images with clear and concise documentation.
+
+You can find the corresponding Lychee image [here](https://fleet.linuxserver.io/image?name=linuxserver/lychee).
+
+> {tip} Please note that we, at LycheeOrg, do not endorse LinuxServer.io.
+We give the reader a link to their websites as they provide our software.
+You are free to use their installation, however we will not be providing you support with their solution.
+
+
+## With Cloudron (Princing)
```
Comment (alex-phillips) [inline docs/md/distributions.md:177]: typo?

Comment (d7415) [reply docs/md/distributions.md:177]:
```suggestion
## With Cloudron (Paid)
```
?

File: docs/md/distributions.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+If you use [Visual Studio Code][3], you can use the [remote ssh extension][4]. It will allow you to directly edit your files in your VM without having to upload and download the files.
+
+[1]: http://lychee.test
+[2]: http://lycheeorg.test
+[3]: https://code.visualstudio.com/
+[4]: https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh
+
+
+## With Linuxserver.io (Free)
```
Comment (d7415) [inline docs/md/distributions.md:164]: Is it worth putting a reminder here that there's an official Docker image too?

File: docs/distributions.html [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+                                            <li><a href="http://laravel.com/docs/7.x/requests">Requests</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/responses">Responses</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/views">Views</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/urls">URL Generation</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/session">Session</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/validation">Validation</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/errors">Error Handling</a></li>
+                                            <li><a href="http://laravel.com/docs/7.x/logging">Logging</a></li>
+                                        </ul>
+                                    </li>
```
Comment (d7415) [inline docs/distributions.html:105]: I know this is generated, but should this section be commented out? Or, really, should any of these many commented lines be here at all?

Comment (ildyria) [reply docs/distributions.html:105]: Nah, I still like to see them because it also explains how the infrastructure works if one wants to dev Lychee. :)

Comment (d7415) [reply docs/distributions.html:105]: It makes more sense in context :) But the others? There's a lot of commented out html there....

Comment (ildyria) [reply docs/distributions.html:105]: I was too lazy to completely remove the rest.

File: docs/md/distributions.md (hunk 2 above)
Comment (ildyria) [reply docs/md/distributions.md:164]: I don't know, we already say it in the main installation page: https://lycheeorg.github.io/docs/#installing-lychee

**Pull request merged**